com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/Accumulation/SubFrameManager.cs

```diff
@@ -67,7 +108,6 @@ void Init(int samples, float shutterInterval)
             m_ShutterInterval = samples > 1 ? shutterInterval : 0;
             m_IsRecording = true;
             m_IsRenderingTheFirstFrame = true;
-            Reset();
```

Comment:
We should keep the Reset here, because it ensures that:
- the first frame of motion blur is correct: we have to discard any previous progress, since it might have happened without motion blur.
- all cameras are in-sync: this is necessary because we cannot have different motion blur progress per-camera (because we manipulate Time.*).

Reply:
Done: I even clear the list here.